### DISCUSSION TITLE
outliers

### DISCUSSION THREAD

**Discussion by sabdul4** [AammarTufail/python_ka_chilla2023#64] [Q&A]
day 13  mein jo ammar bhai ne  jo assignment di kahin solution ni mil rha, uska kisi ko samj aagya h tou helo karde

**Comment by Muhammad-Ali-Butt** [comment]
@sabdul4 baba g ne do assignments de thi aap konsi wali ka pooch rahay ho kindly specify kr do

**Reply by sabdul4** [reply]
Last wali jo conditional argument use krne the sns.lmplot m

**Reply by Muhammad-Ali-Butt** [reply]
mujhay exact answer ni mila uska jaisa baba ne kaha tha k lmplot k **y=** walay argument mai condition lagani but i did find another solution which is in the screenshot
![Capture](https://user-images.githubusercontent.com/108717967/212492009-185653fb-7acd-480e-ae39-276b24b48a41.PNG)

is pooray ss ko dekhay ismay maine comments add kiye ha jo maine kia 

mai idhar waisay b explain kr deta hu basically df2 new variable banaya jismay df assign kia along with the condition k fare<300 aur usi df2 ko then sns.lmplot mai use kr lia and reason df2 ko bananay ki ye ha k df mai data original haalat mai rahay.

**Reply by sabdul4** [reply]
![linecode](https://user-images.githubusercontent.com/117725522/212492356-0aa35bee-dbf1-4bf3-9e34-ba404d10ea9a.jpg)

outliers remove tou keyen h pr line wahin h

**Reply by Muhammad-Ali-Butt** [reply]
> outliers remove tou keyen h pr line wahin pr h 

![with condition](https://user-images.githubusercontent.com/108717967/212492661-f3499376-77d8-40a9-900b-d442c7346752.png)
![without condition](https://user-images.githubusercontent.com/108717967/212492665-0bd6e3d6-8080-4c0a-88fb-10538378f5ac.png)

ye check karay ye with aur without condition ha aur bohat hi slight farq ha kyu k without condition wallay mai outliers sirf 2 thay

**Reply by Muhammad-Ali-Butt** [reply]
also i forgot to mention k with condition mai majority data jo ha wo neechay concentrated ha making 250 k aas paas walay as outliers as well

**Reply by sabdul4** [reply]
![newone](https://user-images.githubusercontent.com/117725522/212492921-5c726aa6-7547-4d76-9ab1-d8f57a9e11aa.jpg)
thanks, bro mene b kardiya jo apne btaya

**Reply by Muhammad-Ali-Butt** [reply]
most welcome bhai glad you found it helpful :)
